gran_source_list.tsv

```diff
@@ -11,7 +11,11 @@ usgs-r/EflowStats	v4.0.2
 GLEON/GLMr	v3.1.1
 ldecicco-usgs/EGRETci	v0.0.8
 usgs-r/smwrBase	1.0.4
-usgs-r/smwrGraphs	1.0.7
+usgs-r/smwrGraphs	v1.0.10
 usgs-r/glmtools	v0.9.3
 usgs-r/powstreams	v0.5.1
+<<<<<<< HEAD
```

Comment:
Looks like you checked in the merge conflict.  This should be cleaned up before merging.